<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>897</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QWidget" name="list_widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="list_table">
         <property name="rowCount">
          <number>2</number>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <row/>
         <row/>
         <column>
          <property name="text">
           <string>白名单</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="rowCount">
          <number>1</number>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <row/>
         <column>
          <property name="text">
           <string>黑名单</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="doubt_widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="doubt_table">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>源MAC</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>目标MAC</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>源IP</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>目标IP</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>协议</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>源端口</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>目标端口</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="connection_widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="connection_table">
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>源MAC</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>目标MAC</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>源IP</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>目标IP</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>协议</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>源端口</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>目标端口</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>15</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>自动刷新</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>7</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string/>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>刷新间隔(秒)</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>30</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="refreshButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>刷新列表</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="killButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>终止连接</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>764</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_load"/>
    <addaction name="action_unload"/>
   </widget>
   <widget class="QMenu" name="option_menu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="action_autoRefresh"/>
    <addaction name="action_handRefresh"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="option_menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>46</width>
     <height>39</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_showConnection"/>
   <addaction name="action_doubt"/>
   <addaction name="action_showList"/>
   <addaction name="action_about"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_autoRefresh">
   <property name="text">
    <string>自动刷新</string>
   </property>
   <property name="toolTip">
    <string>自动刷新网络连接</string>
   </property>
  </action>
  <action name="action_handRefresh">
   <property name="text">
    <string>手动刷新</string>
   </property>
  </action>
  <action name="action_load">
   <property name="text">
    <string>装载驱动</string>
   </property>
   <property name="toolTip">
    <string>装载驱动</string>
   </property>
  </action>
  <action name="action_unload">
   <property name="text">
    <string>卸载驱动</string>
   </property>
   <property name="toolTip">
    <string>卸载驱动</string>
   </property>
  </action>
  <action name="action_showConnection">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icon/connect.png</normaloff>:/images/icon/connect.png</iconset>
   </property>
   <property name="text">
    <string>网络连接</string>
   </property>
   <property name="toolTip">
    <string>查看所有网络连接</string>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="action_showList">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icon/list.png</normaloff>:/images/icon/list.png</iconset>
   </property>
   <property name="text">
    <string>名单管理</string>
   </property>
   <property name="toolTip">
    <string>管理黑白名单</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icon/about.png</normaloff>:/images/icon/about.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
   <property name="toolTip">
    <string>关于程序</string>
   </property>
  </action>
  <action name="action_doubt">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icon/doubt.png</normaloff>:/images/icon/doubt.png</iconset>
   </property>
   <property name="text">
    <string>可疑列表</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
